src/platform_impl/macos/window.rs

```diff
@@ -1236,6 +1236,24 @@ impl WindowExtMacOS for UnownedWindow {
         .setHasShadow_(if has_shadow { YES } else { NO })
     }
   }
+
+  #[inline]
+  fn set_is_document_edited(&self, edited: bool) {
+    unsafe {
+      self
+        .ns_window
+        .setDocumentEdited_(edited)
+    }
+  }
+
+  #[inline]
+  fn is_document_edited(&self) -> bool {
+    unsafe {
+      self
+        .ns_window
+        .isDocumentEdited()
```

Comment:
looks like a new version of cocoa won't be released anytime soon and we have been waiting for ages, can you just inline the objc call from this PR https://github.com/servo/core-foundation-rs/pull/493/files here